flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/batch/table/validation/SortValidationTest.scala

```diff
@@ -27,10 +27,26 @@ import org.junit._
 class SortValidationTest extends TableTestBase {
```

Comment:
It had better add the following two validation tests:
```
 ds.orderBy('a.asc).offset(10).offset(10)
 ds.orderBy('a.asc).fetch(5)
```
Now` fetch` can only be used after `offset`, maybe the `fetch` be used after `orderBy`.  `ds.orderBy('a.asc).fetch(5)` is equivalent with `ds.orderBy('a.asc).offset(0)fetch(5)`

Reply:
Right, I'll add a test that validates that `table.orderBy('a.asc).offset(10).offset(10)` is rejected.

`table.orderBy('a.asc).fetch(5)` is already supported as you suggested (see the modified `SortITCase`).